generic_config_updater/gu_common.py

```diff
@@ -57,8 +57,8 @@ def get_config_db_as_json(self):
         return json.loads(text)
 
     def _get_config_db_as_text(self):
-        # TODO: Getting configs from CLI is very slow, need to get it from sonic-cffgen directly
-        cmd = "show runningconfiguration all"
```

Comment:
>"show runningconfiguration all" [](http://example.com/codeflow?start=14&length=31)

The original cmd is better because it is the formal CLI  interface. Event today the implementation is the same, but in future, the formal CLI will work better with multi-asic.

Instead, you can remove unwanted fields in `get_config_db_as_json` function.